### PR TITLE
helm: add job status consumer prefetch count envar

### PR DESCRIPTION
**Pull request by audrium**
closes https://github.com/reanahub/reana-server/issues/410

Related PR: https://github.com/reanahub/reana-workflow-controller/pull/413

### CODE REVIEW

File: helm/reana/README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_workflow_controller.imagePullPolicy`   | REANA-Workflow-Controller image pull policy                                          | IfNotPresent                                    |
 | `components.reana_workflow_controller.environment.REANA_JOB_HOSTPATH_MOUNTS`   | JSON list of optional hostPath mounts, for all user jobs. Each mount object has a key `name` (name of the mount), `hostPath` (path to the directory to be mounted from the Kubernetes nodes) and `mountPath` (path inside the job containers where the `hostPath` will be mounted)            | None                                            |
 | `components.reana_workflow_controller.environment.REANA_RUNTIME_KUBERNETES_KEEP_ALIVE_JOBS_WITH_STATUSES` | Keep alive Kubernetes user runtime jobs depending on status (`finished` and/or `failed`). | None |
+| `components.reana_workflow_controller.environment.REANA_JOB_STATUS_CONSUMER_PREFETCH_COUNT` | Define max number of messages that can be delivered to the `jobs-status` queue consumer. | 200 |
```
Comment (VMois) [inline helm/reana/README.md:44]: *nitpick*:

```suggestion
| `components.reana_workflow_controller.environment.REANA_JOB_STATUS_CONSUMER_PREFETCH_COUNT` | Define max number of unacknowledged deliveries that are permitted on`jobs-status` queue consumer. | 200 |
```

**Review by VMois — COMMENTED** [review]
LGTM. I did not do manual tests.

One small comment.

**Review by tiborsimko — APPROVED** [review]
Tested with https://github.com/reanahub/reana-workflow-controller/pull/413, works well

File: helm/configurations/nfs-provisioner-values.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# helm repo add stable https://charts.helm.sh/stable
+# helm install reana-dev-storage stable/nfs-server-provisioner -f nfs-provisioner-values.yaml
+# helm uninstall reana-dev-storage
+
+storageClass:
```
Comment (tiborsimko) [inline helm/configurations/nfs-provisioner-values.yaml:5]: @audrium Did this file appear here by mistake? (unrelated change...)

Comment (audrium) [reply helm/configurations/nfs-provisioner-values.yaml]: yep, accidentally slipped through while I was rebasing
